src/midiEditorCommands.cpp

```diff
@@ -1311,6 +1311,33 @@ void postMidiSelectNotes(int command) {
 		count ));
 }
 
+int countSelectedCCs(MediaItem_Take* take) {
+	int evtIndex=-1;
+	int count=0;
+	for(;;){
+		evtIndex = MIDI_EnumSelCC(take, evtIndex);
+		if (evtIndex == -1) {
+			break;
+		}
+		++count;
+	}
+	return count;
+}
+
+void postMidiSelectCCs(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int count = countSelectedCCs (take);
+	if (fakeFocus != FOCUS_NOTE && fakeFocus != FOCUS_CC) {
```

Comment:
I realise that postMidiSelectEvents has this same check. However, I think it probably makes sense to just unconditionally change the focus to CC here. Unless you can think of some reason the focus should stay on notes when selecting CCs?